README.md

```diff
@@ -35,6 +36,14 @@ No other configuration is necessary. Simply **docker build** and
     MAINTAINER John Doe "jdoe@example.org"
     CMD ["config/dev-transactor.properties"]
 
+## CI
+
+This repo is built and tagged automatically by Codefresh.
+- Push to any branch it will tagged with the branch name in lowercase, eg: `642622649254.dkr.ecr.us-west-2.amazonaws.com/docker-datomic:cloud-123`.
```

Comment:
Would you replace the AWS account id with an alias `[parkside-core-services].dkr.ecr.us-west-2....`? The AWS account id is not a secret, but [it may give an advantage to the attackers](https://security.stackexchange.com/questions/192539/keeping-aws-account-id-secret) so we should hide it whenever possible. Only GitHub support can delete the PR and commit, and I think we don't need to request this time.

Reply:
Hi @k2n, just removed it from the README, I force-pushed so that it doesn't show in the commit history, unfortunately, it will still show in your PR comment, maybe if you delete it it will not be visible, thanks